recipes-bsp/u-boot/u-boot-imx-mfgtool_2022.04.bb

```diff
@@ -2,8 +2,6 @@
 # Copyright (C) 2014-2016 Freescale Semiconductor
 # Copyright 2017-2019 NXP
 
-FILESEXTRAPATHS:prepend := "${THISDIR}/u-boot-imx:"
```

Comment:
Really?

Reply:
The trouble comes from including the `u-boot-imx` recipe.

Reply:
But why is it causing problems? Usually, for this kind of recipe, we use the base recipe files as a place for patches and similarly used on both recipes.

If the `u-boot-imx` recipe has a patch file, I believe it will fail to find it.

Reply:
No problems. I dropped it because I misunderstood the deeper reason for the line and thought I was just doing a simple cleanup. I will restore it and add a comment to make the intent clear.